generators/info/index.js

```diff
@@ -38,10 +38,10 @@ module.exports = ReportGenerator.extend({
 
         checkJHipster() {
             const done = this.async();
-            this.log('##### **JHipster Version(s)**');
+            console.log('##### **JHipster Version(s)**');
```

Comment:
Ah sorry, I read your other mail just now. Ok I get the issue. In that case I think you should add an `// eslint-disable-line` comment when using it. else eslint will fail